python/lint/action.yaml

```diff
@@ -0,0 +1,43 @@
+name: 'Lint python project'
+description: 'Runs few python linter'
+inputs:
+  python-version:
+    required: false
+    description: Version of python
+    default: "3.10.x"
+  use-black:
+    required: false
+    description: Use black linter
+    default: true
+  use-flake:
+    required: false
+    description: Use Flake8 linter
+    default: true
+  use-isort:
+    required: false
+    description: Use Isort linter
+    default: true
+runs:
+  using: "composite"
+  steps:
+    - name: Install Python
+      uses: actions/setup-python@v4
+      with:
+        python-version: ${{ inputs.python-version }}
+    - name: Run isort linter
+      if: ${{ inputs.use-isort == 'true' }}
+      uses: jamescurtin/isort-action@master
```

Comment:
I don't think we should use 3rd party actions without pinning it to commit hash and reviewing its codebase..

and speaking of 3rd party - if we use it anyway, isn't there some all-in-one proven-in-action things? like some https://github.com/ricardochaves/python-lint that's wrapping black, pylint, isort, pycodestyle, flake8, mypy?